src/components/page/PrTable.js

```diff
@@ -39,11 +58,57 @@ const PrTable = ({ query }) => {
     {},
     [query]
   );
+  const [prRecords, setPrRecords] = useState({});
+
+  useEffect(() => {
+    let mounted = true;
+    if (!data) return;
+
+    const refNums = [];
+    const prsByRefNum = {};
+
+    for (let pr of data) {
+      const refNum = pr.title.split(" ")[0].toUpperCase();
```

Comment:
Can we also use the branch name (not the PR title) to match?

Reply:
Done, matches on both now, preferring the branch name